### PR TITLE
Replace `JudgmentMediaSummaryFile` model with `AttachedFiles` model

### PR DESCRIPTION
**Pull request by nickmwangemi**
image

### CODE REVIEW

File: peachjam/models/judgment.py [hunk 1]
```diff
@@ -191,18 +191,14 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
 
-class JudgmentMediaSummaryFile(AttachmentAbstractModel):
-    SAVE_FOLDER = "media_summary_files"
+class AttachedFileNature(models.Model):
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:194]: This can move into `peachjam/models/core_document_model.py` since it's no longer judgment-specific.

Comment (longhotsummer) [reply peachjam/models/judgment.py:194]: Ditto for AttachedFiles

File: peachjam/models/judgment.py [hunk 2]
```diff
@@ -191,18 +191,14 @@ def save(self, *args, **kwargs):
...
-    updated_at = models.DateTimeField(auto_now=True)
+    def __str__(self):
+        return self.name
 
-    class Meta:
-        ordering = ["document", "filename"]
 
-    def __str__(self):
-        return f"{self.filename}"
+class AttachedFiles(AttachmentAbstractModel):
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:202]: Do you need to override `SAVE_FOLDER = "attachments"`?

**Comment by longhotsummer** [comment]
This is looking good, Nick. 

The attached files must be added to the metadata block, something like this:

image

File: peachjam/models/judgment.py [hunk 3]
```diff
@@ -204,20 +203,3 @@ def get_case_number_string(self):
     def save(self, *args, **kwargs):
         self.string = self.get_case_number_string()
         return super().save(*args, **kwargs)
-
-
-class JudgmentMediaSummaryFile(AttachmentAbstractModel):
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:209]: Note to self: none of our databases have data for this model so we don't need a migration.

File: peachjam/templates/peachjam/layouts/document_detail.html [hunk 4]
```diff
@@ -145,6 +145,25 @@ <h1 class="py-4">{{ document.title }}</h1>
                           </dd>
                         {% endif %}
                       {% endblock %}
+
+                      {% block document-metadata-media-summary-download %}
+                        {% if document.attachedfiles_set %}
+                          <dt>{% translate 'Attachments' %}</dt>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/layouts/document_detail.html:151]: I'd like this done a bit differently, please. The user shouldn't have to understand our concept of attachments. Rather, show each attached file as its own metadata entry.

```
Language          English
Media summary     Download PDF (3.2MB)
Media summary     Download DOCX (1MB)
Original document Download RTF (1.2MB)
  
```

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Minor change to layout, pls.

**Comment by longhotsummer** [comment]
@actlikewill if https://github.com/laws-africa/peachjam/pull/518 is merged first then this will need to be updated

File: peachjam/models/judgment.py (hunk 3 above)
Comment (nickmwangemi) [reply peachjam/models/judgment.py:209]: @longhotsummer Does that mean I remove the migrations file?

Comment (longhotsummer) [reply peachjam/models/judgment.py:209]: No, the model  migrations are needed, but we don't need to migrate old data (you don't appear to have added that in any case). So It's all good.

File: peachjam/templates/peachjam/layouts/document_detail.html [hunk 5]
```diff
@@ -145,6 +145,33 @@ <h1 class="py-4">{{ document.title }}</h1>
                           </dd>
                         {% endif %}
                       {% endblock %}
+
+                      {% block document-metadata-media-summary-download %}
+                        {% if document.attachedfiles_set %}
+                          {% for attachedfile in document.attachedfiles_set.all %}
+                            {% if attachedfile.nature.name == 'Media summary' %}
+                              <dt>{% translate 'Media summary' %}</dt>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/layouts/document_detail.html:153]: No need to special case anything, just use `attachedfile.nature.name` as the `dt` value.

**Review by longhotsummer — COMMENTED** [review]
Great, looks good. Please coordinate with @actlikewill -- one of you needs to update the now outdated use of JudgmentMediaSummaryFile elsewhere in the code (particularly the importer).

**Review by longhotsummer — APPROVED** [review]

**Comment by nickmwangemi** [comment]
I've updated and tested the new `AttachedFiles` model together with @actlikewill, and the importer works.

**Pull request merged**